<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cloud_homogenization_test_form</class>
 <widget class="QDialog" name="cloud_homogenization_test_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cloud Homogenization Test From</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>441</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="varset_1">
    <item row="0" column="0">
     <widget class="QLabel" name="cloud_in_path_label">
      <property name="text">
       <string>cloud in path</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="cloud_in_ledit"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="cloud_in_browse_btn">
      <property name="text">
       <string>browse...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="cloud_out_path_label">
      <property name="text">
       <string>cloud out path</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="cloud_out_ledit"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="cloud_out_browse_btn">
      <property name="text">
       <string>browse...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>151</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="varset_2">
    <item>
     <widget class="QLabel" name="epsilon_label">
      <property name="text">
       <string>epsilon</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="epsilon_sb">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>299</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="help_btn">
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="launch_test_btn">
      <property name="text">
       <string>Launch test</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
